packages/widgetbook_cli/pubspec.yaml

```diff
@@ -14,18 +14,25 @@ dev_dependencies:
   freezed: ^2.0.3
   json_serializable: ^6.2.0
   lints: ^1.0.0
+  mocktail: ^0.3.0
   very_good_analysis: ^3.0.0
 dependencies: 
   archive: ^3.3.0
   args: ^2.3.0
+  ci: ^0.1.0
   dio: ^4.0.6
   file: ^6.1.2
   freezed_annotation: ^2.0.3
-  widgetbook_git: ^2.1.0 
   http_parser: ^4.0.1
+  interact: ^2.1.1
+  mason_logger: ^0.1.3
   path: ^1.8.2
+  pub_updater: ^0.2.2 
+  pubspec: ^2.3.0
```

Comment:
looks ok to me, not sure if this package is well maintained

Reply:
Yes seems like ,  last  review was
Published [3 months ago](https://pub.dev/packages/pubspec)